src/core/entities/MRVideoEntity.js

```diff
@@ -59,7 +59,8 @@ export class MRVideoEntity extends MRMediaEntity {
      * @function
      * @description Callback function of MREntity - handles setting up this video and associated 3D geometry style (from css) once it is connected to run as an entity component.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?